Source/Dafny/Resolver.cs

```diff
@@ -446,9 +446,13 @@ enum ValuetypeVariety {
       rewriters.Add(new InductionRewriter(reporter));
       rewriters.Add(new PrintEffectEnforcement(reporter));
 
+      var builtInRewritersCount = rewriters.Count;
+      var pluginRewriters = new List<IRewriter>() { };
```

Comment:
What is the purpose of introducing the `pluginRewriters` variable? Is it just for clarity? I don't see it used after line `454` but maybe I'm missing something.

Reply:
Right, since I'm now only taking `builtInRewritersCount`, I can remove these extra variables.